plugin/fzf.vim

```diff
@@ -392,12 +391,12 @@ try
   for tf in values(a:temps)
     silent! call delete(tf)
   endfor
-
-  return lines
 catch
   if stridx(v:exception, ':E325:') < 0
     echoerr v:exception
   endif
+finally
+  return lines
```

Comment:
When exception is caught, empty string is returned, which causes problem [here](https://github.com/junegunn/fzf/pull/506/files#diff-24d9d57e35df5afce7890f2400917919R349) because `ret` is initialized to a list.